lib/authentication.js

```diff
@@ -0,0 +1,147 @@
+'use strict';
+
+const MAX_PASS_LENGTH = 128;
+const MIN_PASS_LENGTH = 10;
+const MIN_PHRASE_LENGTH = 20;
+const MIN_OPTIONAL_TESTS = 3;
+
+const MIN_LOGIN_LENGTH = 3;
```

Comment:
Are there recommendations to have 3 letter logins? I think we need at least 6 chars but we need to follow recommendations if any standardization authority have it?